### PR TITLE
[candidate_parameters] Update testing plan info of permission role needed

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Updates testing plan to let the user testing know they need the role of data entry in permissions.

#### Link(s) to related issue(s)

https://github.com/aces/Loris/issues/6275

### CODE REVIEW

File: modules/candidate_parameters/test/TestPlan.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 1. Check Permissions:
     * _candidate_parameter_view_ (_View Candidate Parameters_) permission (you should not be able to edit fields)
 	* _candidate_parameter_edit_ (_Edit Candidate Parameters_) permission (you should be able to edit fields)
+	* The user role in permissions should be set to Data entry.
```
Comment (driusan) [inline modules/candidate_parameters/test/TestPlan.md:6]:
```suggestion
	* The user should have the Data Entry permission
```

**Comment by ridz1208** [comment]
@driusan during one of our meetings we said that we should remove the need for data-entry permission alltogether. I had updated the issue. do you still wanna go down that road ?

**Comment by driusan** [comment]
@ridz1208 That's true, I just suggested the change to the wording because it's clearer and more consistent to how the permission is referred to elsewhere.

**Comment by ridz1208** [comment]
@driusan sos should we close this PR and change the permission requirements instead ?

**Comment by driusan** [comment]
I have no strong feelings on whether we should do that immediately to fix #6275 or update the test plan, as long as the code and test plan are consistent.

**Comment by driusan** [comment]
@maltheism can you update the code as @ridz1208 suggested instead of updating the test plan?

**Comment by maltheism** [comment]
Closing this PR in favour of https://github.com/aces/Loris/pull/6398

**Pull request closed without merging**